microsoft-edge/progressive-web-apps-chromium/how-to/window-controls-overlay.md

```diff
@@ -88,14 +88,16 @@ You can use these environment variables to position and size your app's title ba
 
 ```css
 #title-bar {
-    position: absolute;
+    position: fixed;
     left: env(titlebar-area-x);
     top: env(titlebar-area-y);
     height: env(titlebar-area-height);
     width: env(titlebar-area-width);
 }
 ```
 
+Using `position:fixed;` makes sure your title bar does not scroll with the rest of the content and instead stays aligned with the window controls overlay.
```

Comment:
```suggestion
Using `position: fixed;` makes sure your title bar does not scroll with the rest of the content and instead stays aligned with the window controls overlay.
```

Reply:
Also, would it help to say "... fixed;` rather than `position: absolute;` ..." to make it more clear, because the old setting will no longer be visible?

Reply:
I'm not sure, because fixed and absolute are not the only 2 choices. So it's not a question of using either one or the other. So it feels odd to me to compare only to absolute, and not static, relative, and sticky for example.